codebase/superdarn/src.bin/tk/tool/make_grid.2.0/make_grid.c

```diff
@@ -7,6 +7,9 @@
 
  Copyright (c) 2012 The Johns Hopkins University/Applied Physics Laboratory
 
+ Modifications:
+ 2021-01-11 Marina Schmidt fixing concatenation bug with start time
```

Comment:
Mention the other changes here too?
```suggestion
 2021-01-11 Marina Schmidt fixing concatenation bug with start/end time, and deprecated -c flag
```